crates/libm-test/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "libm-test"
+version = "0.1.0"
+authors = ["Benjamin Schultzer <benjamin@schultzer.com>"]
+
+[lib]
+proc-macro = true
```

Comment:
You probably want to separate the library that does the tests from the proc macro that just generates tests and calls into that library.